lalrpop/src/normalize/mod.rs

```diff
@@ -51,6 +51,11 @@ fn lower_helper(session: &Session, grammar: pt::Grammar, validate: bool) -> Norm
             prevalidate::validate(&grammar)?;
         }
     );
+    let grammar = profile!(
+        session,
+        "Grammar preprocessing",
+        preprocess::preprocess(session, grammar)?
```

Comment:
I'm not sure "preprocess" is the best name for this.  Really, the whole normalize step is preprocessing.  How does "expand_cfg" sound as alternative?  I'm not tied to that name at all, just brainstorming for a more specific name.